web_development_101/tying_it_all_together/conclusion.md

```diff
@@ -4,8 +4,14 @@ You should be fairly comfortable typing commands into your computer's command li
 
 With what you know already, you could start putting together some basic websites or strike out to increase your knowledge on your own.  However, we're here to help you get a lot further than that, to the point where you can confidently build a full-featured and scalable web application, whether on your own or working as a developer in a top-tier tech company.  
 
-You've come a long way up the learning curve but there is so much more cool stuff that you'll learn to do!
+You've come a long way up the learning curve but there is so much more cool stuff that you'll learn to do, and now is the time to decide which path you want to take. Either Ruby with Ruby on Rails or JavaScript with Node/Express. Both paths will guide you in building skills, building up a portfolio of projects and become a great developer. There is no right answer here on what is the next best path. Learning a differnet language can help you become a more well rounded developer, but sticking with one the whole way though could make you more of an expert in the lanaguage itself. Both paths will help you become a great developer. The important thing is to not switch halfway though, this will only set you back.
 
-Buckle up, because this is only the beginning. Now it's time to delve deeper into the areas covered by Web Dev 101. There are some fantastic projects in the courses that follow which will challenge you to the limit, but you'll come out the other side ready to go and make your mark in the programming world.
+Learning how to program is **much** more than just learning a language. It's about learning how to research and solve difficult problems, build an engineering mindset, work on complex problems and codebases, communicate effectively with other developers, and more. Luckly all these skills are transferrable to other languages and domains of software and because most languages are similar in concept and structure, it will be easy to move into a new language with far less difficulty than learning your first one. 
+
+Arguments about learning the languages used in your area may sound convincing, but because a _good_ developer can transfer skills easily, what you learn isn't going to change outcomes of getting a job much. Though, if you cannot shake that fear, do whatever you need to just stay on the train and finish the course. 
```

Comment:
"do whatever you need to just stay on the train and finish the course."
->
"do whatever you need to do just to stay on the train and finish the course."